mods/action.js

```diff
@@ -155,6 +155,9 @@ ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
     body.action = defBody.action;
     body.format = body.format || defBody.format || 'json';
     body.formatversion = body.formatversion || defBody.formatversion || 1;
+    if (defBody.rawcontinue !== undefined) {
```

Comment:
In case of req.body.continue is provided the response is returned in another format we cannot parse, so we need raw continue either way